docs/usage-as-operator.md

```diff
@@ -22,17 +19,29 @@ countUpdateDomains:
 countFaultDomains:
 - region: westeurope
   count: 3
+acceleratedNetworkingMachineTypes:
+- Standard_D3_v2
```

Comment:
```suggestion
- name: Standard_D3_v2
```

Reply:
Hmm yes, we could do that, but it is simply a list of machine type names. We could rename the list key to `acceleratedNetworkingMachineTypeNames` to make it more explicit.

Reply:
Today it's a list of machine types. Tomorrow it's maybe more (e.g., additional configuration/information per machine type). Just guessing, of course, though, generally it's API-wise a good practice to plan for such future enhancements. If you also know that something like this is highly unlikely to happen then keep `[]string`, though.

Reply:
Hmm yes I agree with you. Then I would suppose to make it even more generic similar we have it for the cloud provider specific information for images.
Let's have a list in the provider section like that.
```
providerConfig:
  machineTypes:
  - name: Standard_XXX
    acceleratedNetworking: true
    premiumStorage: true # Some real property which would make sense for validation purposes.
    someOtherProperty: xxx
  - name: Standard_YYY
    acceleratedNetworking: false
  - name: Standard_ZZZ
```

Reply:
Sure, sounds great 👍